src/regenerate.rs

```diff
@@ -120,7 +122,7 @@ fn get_corresponding_type(elements: &[(Option<TypeStruct>, Vec<String>)],
 // The hashmap key is `Some(file name)` or `None` for entries that ignore file name
 pub fn regenerate_comments(work_dir: &Path, path: &str,
         infos: &mut HashMap<Option<String>, Vec<(Option<TypeStruct>, Vec<String>)>>,
-        ignore_macros: bool) {
+        ignore_macros: bool, ignore_doc_commented: bool) {
```

Comment:
Just look at the tests: `false, false` or at [this](https://github.com/gtk-rs/lgpl-docs/blob/master/src/lib.rs#L26). I could add comments there of course but we could easily have something more structured as well.